src/routes/RequestQueueRoute.test.js

```diff
@@ -0,0 +1,219 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import RequestQueueRoute from './RequestQueueRoute';
+
+jest.mock('@folio/stripes/core', () => ({
+  stripesConnect: jest.fn(Component => props => <Component {...props} />)
+}));
+
+jest.mock('react-router-prop-types', () => ({
+  location: jest.fn(),
+}));
+
+jest.mock('../views/RequestQueueView', () => {
+  return function MockRequestQueueView(props) {
+    return (
+      <div>
+        <p>Mock RequestQueueView</p>
+        <button type="button" onClick={props.onClose}>Close</button>
+      </div>
+    );
+  };
+});
+
+jest.mock('../utils', () => ({
+  getTlrSettings: jest.fn(() => ({ titleLevelRequestsFeatureEnabled: false })),
+  isPageRequest: jest.fn(),
+}));
+
+const mockResources = {
+  request: {
+    records: [{
+      id: '1',
+      holdingsRecordId: '2',
+      itemId: '3',
+    }],
+  },
+  holdings: {
+    records: [{
+      id: '123',
+      callNumber: 'ABC123',
+    }]
+  },
+  items: {
+    records: [{
+      id: '3',
+    }],
+  },
+  configs: {
+    records: [{
+      value: {
+        titleLevelRequestsFeatureEnabled: true,
+      },
+    }],
+    hasLoaded: true,
+  },
+  requests: {
+    records: [{
+      shouldRefresh: false,
+    }],
+  },
+};
+
+const mockMutator = {
+  reorderInstanceQueue: {
+    POST: jest.fn(),
+  },
+  reorderItemQueue: {
+    POST: jest.fn(),
+  },
+  requests: {
+    reset: jest.fn(),
+    GET: jest.fn(),
+  },
+};
+
+const mockHistory = {
+  push: jest.fn(),
+  goBack: jest.fn(),
+};
+const mockMatch = {
+  params: {
+    id: '2',
+    requestId: '1',
+  },
+};
+
+const mockLocation = {
+  state: {
+    request: {
+      id: '1',
+    },
+  },
+};
+
+describe('RequestQueueRoute', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+  it('renders without error', () => {
+    const { container } = render(
+      <RequestQueueRoute
+        resources={mockResources}
+        mutator={mockMutator}
+        history={mockHistory}
+        location={mockLocation}
+        match={mockMatch}
+      />
+    );
+    expect(container).toBeInTheDocument();
+  });
+  it('should call setTlrSettings on mount if configs have loaded', () => {
```

Comment:
Name of the test does not fit test body.
In this test you check that `requests.GET` is called after rerender and is not called after mount.

Reply:
Name of the test does not fit test body.
In this test you check that requests.GET is called after rerender but not after the first render (mount).